src/CodeGeneratorGo.cs

```diff
@@ -17,7 +17,6 @@ public class CodeGeneratorGo : CodeGenerator
         private const string ModelsFileName = "models";
         private const string ClientFileName = "client";
         private const string VersionFileName = "version";
-        private const string MarshallingFileName = "marshalling";

```

Comment:
>MarshallingFileName [](start = 29, length = 19)

note that I removed the marshalling file from the output. it seems most of it should go to the models file

Reply:
The reason for this file was to separate the exported types from the non-exported types/implementation details (i.e. the content of marshalling.go is interesting to us but not to consumers of the SDK).  If merging this into the models file makes it easier to generate the content then we can leave it merged.

Reply:
makes sense, but I think to keep the generator code cleaner its easier to put them in the models also would prefer to have all the publicly used types ( and their marshallers ) to be in the models.